python/ray/function_manager.py

```diff
@@ -404,11 +404,7 @@ def f():
                 self._worker,
                 ray_constants.REGISTER_REMOTE_FUNCTION_PUSH_ERROR,
                 traceback_str,
-                driver_id=driver_id,
```

Comment:
Can we replace `traceback_str` with

`"Failed to unpickle the remote function '{}' with function ID {}. Traceback:\n{}".format(function_name, function_id.hex(), traceback_str)`

and make sure that the error looks reasonable?